src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -16,38 +16,56 @@ using namespace ssvuj;
-    // can be used to create malware and is capable of destroying computers.
-    lua.clearVariable("os");
-
-    // This destroys the "io" library completely. This may not be a permanent
-    // action at the moment, but this is the best solution we have at the
-    // moment. This library is dedicated to manipulating files and their
-    // contents, which can be used maliciously.
-    lua.clearVariable("io");
-
-    // This destroys the "debug" library completely. The debug library is next
-    // to useless in Open Hexagon (considering we have our own methods of
-    // debugging), and it allows people to access destroyed modules with the
-    // getregistry function.
-    lua.clearVariable("debug");
-
-    // This function allows pack developers to set the seed in Lua. This
-    // function breaks replays. Can be removed once this is handled properly.
-    lua.clearVariable("math.randomseed");
-
-    // These functions are being deleted as they can assist in restoring
-    // destroyed modules. However, we cannot destroy the whole library as
-    // the other functions are needed for the "require" function to work
-    // properly.
-    lua.clearVariable("package.loadlib");
-    lua.clearVariable("package.searchpath");
-}
+    void HexagonGame::destroyMaliciousFunctions()
+    {
+        // This destroys the "os" library completely. This library is capable of
+        // file manipulation, running shell commands, and messing up the replay
+        // system completely. os.execute(), one of the functions in this library,
+        // can be used to create malware and is capable of destroying computers.
+        lua.clearVariable("os");
+
+        // This destroys some of the "io" functions completely.
+        // This library is dedicated to manipulating files and their contents,
+        // which can be used maliciously.
+        lua.clearVariable("io.popen");
+        lua.clearVariable("io.flush");
+        lua.clearVariable("io.write");
+        lua.clearVariable("io.setvbuf");
+
+        // This destroys the "debug" library completely. The debug library is next
+        // to useless in Open Hexagon (considering we have our own methods of
+        // debugging), and it allows people to access destroyed modules with the
+        // getregistry function.
+        lua.clearVariable("debug");
+
+        // This function allows pack developers to set the seed in Lua. This
+        // function breaks replays. Can be removed once this is handled properly.
+        lua.clearVariable("math.randomseed");
+
+        // These functions are being deleted as they can assist in restoring
+        // destroyed modules. However, we cannot destroy the whole library as
+        // the other functions are needed for the "require" function to work
+        // properly.
+        lua.clearVariable("package.loadlib");
+        lua.clearVariable("package.searchpath");
+
+        // Synth Morxemplum (c) 2020
```

Comment:
We can't have copyright notices in the middle of the code. The project is licensed under AFL3.0, and all contributors need to be credited. I am doing that in the README, and will also do that in-game.

Reply:
that was a joke related thing, sorry lol.